### DISCUSSION TITLE
美國餐飲業嚴重缺工，麥當勞推出首家全自動得來速

### DISCUSSION THREAD

**Discussion by Yilin20021016** [1101-GS3073-C/Python-Community-GS3073-C#671] [General]
作為一個社恐我需要這個東西
https://technews.tw/2022/12/23/mcdonalds-corporation-fully-automatic-drive-thru/

**Comment by xCk27x** [comment]
[麥當勞首家無人店開業 「恐致數百萬工作機會消失」](https://businessfocus.io/article/219748/%E7%BE%8E%E5%9C%8B-%E9%BA%A5%E7%95%B6%E5%8B%9E-%E8%87%AA%E5%8B%95-%E5%BE%97%E4%BE%86%E9%80%9F)

- 這篇文章裡說到 其實這家全自動化麥當勞還是需要雇用員工的 大家先不用擔心
- 只是原本在櫃台和外場的員工被轉移到內場 出餐的部分仍由人類員工負責

> 但如果要我在一家看不見人的餐廳點餐 我會覺得很可怕QQ

### 究竟這種用機器取代服務業的做法會不會受到歡迎呢?

我是覺得點餐時的交流算蠻必須的環節啦 畢竟一整天能跟別人講話的機會不多
還是需要從社交中獲得一點生命的實感才行

**Comment by bww0401** [comment]
我覺得全自動在疫情嚴峻時刻非常需要，可以減少病毒傳遞的風險，但如果喜歡人與人的溫度的話，應該還是可以去內用吧，這樣的作法可以應付更多不同需求的人，我認為是一個創舉

**Comment by skyling0299** [comment]
華爾街日報報導，白宮首席經濟顧問Brian Deese 20日受訪時表示，提供優質、負擔得起的托兒服務實際上有助減輕家庭負擔，進而讓更多人可以重返職場。

美國勞工部前首席經濟學家Harry Holzer表示，當前的勞動力短缺程度是他一生中從未見過的。Holzer說，拜登政府正設法讓更多人進入職場，美國勞動力市場一直缺乏的是年輕女性。

但為甚麼是年輕女性

**Comment by liangyinglly** [comment]
沒想到美國休閒餐飲業職位目前這麽短缺，但那些科技大公司好像都在裁員ㄟ？

**Comment by yjpxi** [comment]
但是這樣的系統也可能面臨一些挑戰，如設備維護、技術故障的處理、以及如何在技術和人力之間找到平衡，會不會造成有些人找不到工作。
